lib/date.nit

```diff
@@ -0,0 +1,111 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Date manipulation
+module date
+
+# The main class Date
+public class Date
+
+	private var years_In: Int 
+	private var mounth_In: Int
+	private var day_In: Int
+
+	private var hours_In: nullable Int = 0
+	private var min_In: nullable Int = 0
+	private var sec_In: nullable Int = 0
```

Comment:
Also, the `I` in `hours_In` and others should be lowercase. But actually, you could simplify the names to only `hours`, etc.

This is a design concern but all the attributes could be public (so they are readable). They can be useful to users and you wouldn't need the getters. By default, an attribute is readable but not writable from outside the module.